pageserver/src/http/openapi_spec.yml

```diff
@@ -251,6 +251,13 @@ paths:
             type: string
             format: date-time
           description: A timestamp to get the LSN
+        - name: with_lease
+          in: query
+          required: false
```

Comment:
The flag is off by default until the control plane enables it in their API calls.